deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
@LaurentDumont - wondering if we can get more information about the issue this *might* cause? We are noticing some unexpected behavior when having this label defined.